src/components/CreateArticleForm.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import axios from "axios";
+import {Form, Input, TextArea, Button, Message } from 'semantic-ui-react'
+
+const CreateArticleForm = () => {
+  const [message, setMessage] = useState();
+  const [title, setTitle] = useState();
+  const [subtitle, setSubtitle] = useState();
+  const [content, setContent] = useState();
+
+  let headers = useSelector((state) => state.currentUser);
+  const saveArticle = async (e) => {
+    e.preventDefault();
+    setTitle(e.target.title.value);
+    setSubtitle(e.target.input_sub_title.value);
+    setContent(e.target.input_content.value);
+    setMessage("Your article was created");
```

Comment:
use the status message you get form the backend

Reply:
We set response.data.message for this one too